src/refract/JsonSchema.cc

```diff
@@ -400,394 +400,393 @@ namespace
         addType(schema, TYPE_NAME);
 
         ObjectSchema result{};
-        if (e.empty())
+        if (e.empty()) {
```

Comment:
The only change I made was adding `{` here, replacing the `else` with `} else {`, and then adding `}` on line 414 and then running clang-format on the file.